test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -202,319 +202,6 @@ Feature: Bind values from a secret referred in backing service resource
-                  - name: backend-operator
-                    spec:
-                      replicas: 1
-                      selector:
-                        matchLabels:
-                          name: backend-operator
-                      strategy: {}
-                      template:
-                        metadata:
-                          labels:
-                            name: backend-operator
-                        spec:
-                          containers:
-                          - command:
-                            - backend-operator
-                            env:
-                            - name: WATCH_NAMESPACE
-                              valueFrom:
-                                fieldRef:
-                                  fieldPath: metadata.annotations['olm.targetNamespaces']
-                            - name: POD_NAME
-                              valueFrom:
-                                fieldRef:
-                                  fieldPath: metadata.name
-                            - name: OPERATOR_NAME
-                              value: backend-operator
-                            image: REPLACE_IMAGE
-                            imagePullPolicy: Always
-                            name: backend-operator
-                            resources: {}
-                strategy: deployment
-            """
-        And The Secret is present
-            """
-            apiVersion: v1
-            kind: Secret
-            metadata:
-                name: $scenario_id-secret
-            stringData:
-                username: AzureDiamond
-                password: hunter2
-            """
-        And The Custom Resource is present
-            """
-            apiVersion: bar.example.com/v1
-            kind: Backend
-            metadata:
-                name: $scenario_id-backend
-            spec:
-                image: docker.io/postgres
-                imageName: postgres
-                dbName: db-demo
-            status:
-                data:
-                    dbCredentials: $scenario_id-secret
-            """
-        When Service Binding is applied
-            """
-            apiVersion: binding.operators.coreos.com/v1alpha1
-            kind: ServiceBinding
-            metadata:
-                name: $scenario_id-binding
-            spec:
-                bindAsFiles: false
-                services:
-                  - group: bar.example.com
-                    version: v1
-                    kind: Backend
-                    name: $scenario_id-backend
-                application:
-                    name: $scenario_id
-                    group: apps
-                    version: v1
-                    resource: deployments
-            """
-        Then Service Binding is ready
-        And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
-        And The application env var "BACKEND_PASSWORD" has value "hunter2"
-
```

Comment:
Was not expecting to remove all these test, just disable them, in case we need to bring OLM Descriptor feature back. @pmacik and @sadlerap how easy would be to bring this back if we need to?

Reply:
It's just a `git revert` away.